config/database.yml

```diff
@@ -0,0 +1,86 @@
+# PostgreSQL. Versions 9.3 and up are supported.
+#
+# Install the pg driver:
+#   gem install pg
+# On macOS with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On macOS with MacPorts:
+#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem "pg"
+#
+default: &default
+  adapter: postgresql
+  encoding: latin1
```

Comment:
- [ ] Please use the `correct` encoding value for your PostgreSQL database configuration. I noticed that while creating the database tables, I got the error and you changed the default setting or configuration of the database. Kindly use `Unicode` instead of latin1, because Unicode encoding is more flexible and has more support for different systems.

![Screenshot from 2024-01-08 16-37-57](https://github.com/Mesi21/talkative/assets/66476329/3b9d763a-cbbd-4e88-a1be-6e22321f5093)